test/.eslintrc.json

```diff
@@ -29,6 +29,9 @@
     "jsdoc/require-returns-type": "off",
     "jsdoc/valid-types": "off",
 
+    // Since we use ts-node we should always require the TS code
+    "no-restricted-modules": ["error", { "patterns": ["**/../lib/**"] }],
```

Comment:
The pattern is `../lib` otherwise it matched a `mocha/lib` and `bson/lib` import we use